.github/workflows/ci-go.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: v1.20.x
```

Comment:
Can we change this to take go version from go.mod? You can checkout what we did in KubeArmor PR https://github.com/kubearmor/KubeArmor/pull/1663